src/app/combo/combo.sample.html

```diff
@@ -106,6 +106,43 @@ <h1 class="combo-sample__title">Remote Binding</h1>
         </igx-simple-combo>
     </section>
 
+    <section class="combo-sample__section">
+        <h1 class="combo-sample__title">Open simple combo on clear by handling the selectionChanging event</h1>
+        <!-- Simple combo with selectionChanging event -->
+        <igx-simple-combo  
+                    #openOnClearSimpleComboEvent
+                    [displayDensity]="'comfortable'"
+                    [(ngModel)]="selectedValueSimpleComboEvent"
+                    type="border"
+                    [data]="items"
+                    [displayKey]="valueKeyVar"
+                    [disabled]="isDisabled"
+                    [valueKey]="valueKeyVar"
+                    [groupKey]="valueKeyVar ? 'region' : ''" 
+                    (selectionChanging)="openOnClearSelectionChanging($event)">
+                <label igxLabel>States</label>
+                <igx-hint>Click clear icon.</igx-hint>
```

Comment:
No need to have a hint here, it's pretty self-explanatory what this sample is trying to show - same for the other one.